cfg_mgmt/reporting.py

```diff
@@ -30,6 +30,60 @@ class CfgElementStatusReport:
     status: typing.Optional[cmm.CfgStatus]
 
 
+def analyse_cfg_element_status(
```

Comment:
naming-nit: I would consider this operation to be more adequately described by either `determine`, or `calculate`